### PR TITLE
SCRUM-7-Converting-Roman-Numerals-to-Integer

### PR DESCRIPTION
**Pull request by mahidharreddyg**
Given a string of Roman Numerals, the program will convert it to an integer value.

### CODE REVIEW

File: DSA _Task/Mahidhar/Roman_Numerals_Conversion.py
Comment (Tanujafule98) [inline DSA _Task/Mahidhar/Roman_Numerals_Conversion.py:1]: this code is giving error. I check this code in multiple compiler. Please check this once again and do the needful modification.

**Review by Tanujafule98 — COMMENTED** [review]
this code is giving error. I check this code in multiple compiler. Please check this once again and do the needful modification.

**Comment by mahidharreddyg** [comment]
Hello Tanuja,

The code I have requested for a pull request doesn't give any errors. I
have attached the output images, both from VS Code and online compiler for
your reference. Please check it and let me know.

Thank you,
Best regards,
Mahidhar Reddy G




[image: image.png]
[image: image.png]

**Comment by Tanujafule98** [comment]
Your code is not running. I checked it. please send me images of the output through email.

**Comment by Tanujafule98** [comment]
images not working that you send in the previous comment. any ways i am approving your pull request.

**Comment by mahidharreddyg** [comment]
> images not working that you send in the previous comment. any ways i am approving your pull request.

Not the comment, please check you mail. Only after that approve the pull request.

**Review by Tanujafule98 — APPROVED** [review]

**Pull request merged**